.github/workflows/upload-and-deploy.yaml

```diff
@@ -88,10 +238,53 @@ jobs:
           Payload: '{"RequestType": "Create"}'
           LogType: Tail
 
+
+  integration-test-develop:
```

Comment:
Did you check that this runs through the pipeline as expected? No failures and you can read the resulting parquet?

Reply:
Yes, I finally got the integration test part to actually trigger the lambda. The only thing is I'm `sam local` building and invoking our lambda. I'm not sure if we want to trigger our lambda directly using `aws invoke` or some github action that we already use? In order to do so, have to update our s3 to glue lambda template and stack configs because we currently don't have a set s3 to glue lambda function name